pkgs/development/libraries/libamqpcpp/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+    name = "libamqpcpp-2.7.4";
+
+    src = fetchurl {
```

Comment:
fetchFromGitHub may be preferable

Reply:
This looks much more suitable - thank you!
It appears to download the same file, but calculates a different hash for it; is this expected?